docs/source/examples/learning-depolarizing-noise.myst

```diff
@@ -0,0 +1,208 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3.8.13 64-bit ('venv_mitiq_learnrepspec')
+  language: python
+  name:  python3
+---
+
+# Learning quasiprobability representations with a depolarizing noise model
+
++++
+
+In this example, we demonstrate the workflow of learning quasiprobability representations of a `CNOT` gate with a depolarizing noise model, from Clifford circuit data. The depolarizing noise model is parameterized by the noise strength, `epsilon`. The resulting quasiprobability representations of the `CNOT` gate are then used to obtain an error-mitigated expectation value with Mitiq's {ref}`probabilistic error cancellation module <guide/pec/pec>`.
+
+```{code-cell} ipython3
+import numpy as np
+from matplotlib import pyplot as plt
+from cirq import (
+    CXPowGate,
+    DepolarizingChannel,
+    Rx,
+    Rz,
+    I,
+    X,
+    Y,
+    Z,
+    LineQubit,
+    Circuit,
+    ops,
+    unitary,
+    InsertStrategy,
+)
+from mitiq import Executor, Observable, PauliString
+from mitiq.interface.mitiq_cirq import compute_density_matrix
+from mitiq.cdr import generate_training_circuits
+from mitiq.cdr._testing import random_x_z_cnot_circuit
+from mitiq.pec import execute_with_pec
+from mitiq.pec.representations.depolarizing import (
+    represent_operation_with_local_depolarizing_noise,
+)
+from mitiq.pec.representations.learning import (
+    biased_noise_loss_function,
+    learn_depolarizing_noise_parameter,
+)
+```
+
+Create a simple Rx-Rz-CNOT circuit using `mitiq.cdr._testing.random_x_z_cnot_circuit`. Here we use an (optional) seed for reproducibility.
+
+```{code-cell} ipython3
+circuit = random_x_z_cnot_circuit(
+    LineQubit.range(2), n_moments=5, random_state=np.random.RandomState(1)
+)
+print(circuit)
+```
+
+In this example, we will learn the strength of depolarizing noise of the `CNOT` gate in the circuit.
+
+```{code-cell} ipython3
+CNOT_ops = list(circuit.findall_operations_with_gate_type(CXPowGate))
+index = CNOT_ops[0][0]
+op = CNOT_ops[0][1]
+operations_to_learn = [Circuit(op)]
```

Comment:
```suggestion
CNOT_op = next(circuit.findall_operations_with_gate_type(CXPowGate))
moment_index, op, _ = CNOT_op
operations_to_learn = [Circuit(op)]
```
WDYT of this?

I was a little lost when I got down to `noisy_execute` because I didn't remember what `op` and `index` were. I wonder if this code should be moved into the same cell as that one, so it's more fresh in the readers mind.

Reply:
At one point I was using the `next` function here, and I forget why I changed it. It should still work and it looks nicer.